src/app/shared/widgets/treatments-list/treatments-list.component.ts

```diff
@@ -0,0 +1,30 @@
+import { Component, OnDestroy, OnInit } from '@angular/core';
+import { Subscription } from 'rxjs';
+import { SentResourceTypes } from '../../../_constants/common';
+import { SendResourceService } from '../../../_services/send-resource.service';
+
+@Component({
+  selector: 'app-treatments-list',
+  templateUrl: './treatments-list.component.html',
+  styleUrls: ['./treatments-list.component.scss'],
+})
+export class TreatmentsListComponent implements OnInit, OnDestroy {
+  treatments: any[] = [];
+  subscription: Subscription = new Subscription();
+
+  constructor(private sendResourceService: SendResourceService) {}
+
+  ngOnInit(): void {
+    this.subscription = this.sendResourceService.getResource().subscribe((response: any) => {
```

Comment:
Please use the array subscription method that we are using throught the project to unsubscribe : example, like it is done in the case-filters.component.ts

Reply:
Done.